File: cases/taint/go/local-overwrite-negative/local_overwrite_negative.go
package dataflowbench

func dfb_source() int { // DFB-SOURCE: local-overwrite-input
	return 1
}

func dfb_sink(value int) {} // DFB-SINK: local-overwrite-sink

func run() {
	value := dfb_source()
	value = 0 // DFB-KILL: local-overwrite-clean
	dfb_sink(value)
}

File: cases/taint/c/local-overwrite-negative/local_overwrite_negative.c
int dfb_source(void) { // DFB-SOURCE: local-overwrite-input
    return 1;
}

void dfb_sink(int value) {} // DFB-SINK: local-overwrite-sink

void run(void) {
    int value = dfb_source();
    value = 0; // DFB-KILL: local-overwrite-clean
    dfb_sink(value);
}
